components/card/card.css

```diff
@@ -0,0 +1,89 @@
+.container {
+  width: 100%;
+  margin: 2rem auto;
+  padding: 2rem;
+  border-radius: 6px;
+  box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
```

Comment:
This box-shadow seems to repeated on other pages as well, try to add a variable for the same.